.buildkite/premerge.steps.yaml

```diff
@@ -25,14 +25,66 @@ common: &common
 # These are then relied on to have stable names by other things, so once named, please beware renaming has consequences.
 
 steps:
-  - label: "build :bash:"
-    command: bash -c ci/build-test.sh
-    <<: *common # This folds the YAML named anchor into this step. Overrides, if any, should follow, not precede.
-    artifact_paths:
-      - logs/**/*
   - label: "verify docs only changes :cop:"
     command: bash -c ci/docs-build-verification.sh
     <<: *common # This folds the YAML named anchor into this step. Overrides, if any, should follow, not precede.
+
   - label: "lint docs :lint-roller:"
     command: bash -c ci/docs-lint.sh
     <<: *common
+
+  - label: "test"
+    command: bash -c ci/test.sh
+    <<: *common
+    artifact_paths:
+      - logs/**/*
+
+  - label: "build :android:"
+    command: bash -c ci/build.sh
+    <<: *common
+    artifact_paths:
+      - logs/**/*
+    env: 
+      WORKER_TYPE: "AndroidClient"
```

Comment:
Curious why do we pass this through env vars not command arguments?

Reply:
It gets very ugly due to `bash -c`. See https://github.com/spatialos/gdk-for-unity/blob/617b98dbcff3618b32c8326ba6d8abde7ba9df14/.buildkite/premerge.steps.yaml for an example of what it would look like.

TL;DR - Parameters passed in the `command` field are taken as parameters to the `bash` command, not the invocation of the script. So you must interpolate the arguments from bash into the invokation of the script. This also means you have to do some nasty YAML escaping to get everything running nicely & edit`trigger.sh` to turn off Buildkite trying to interpolate the variables (which is does when you call `buildkite-agent upload pipeline`).

Actually that wasn't very TL;DR 😬